server/zanata-frontend/src/app/components/TMX/TMXExportModal.js

```diff
@@ -105,16 +105,18 @@ class TMXExportModal extends Component {
       const downloadTMX = handleExportTMX.bind(undefined,
         srcLang.localeDetails, project, version)
       return (
-        <Row type="flex" justify="center" key={localeId}>
+        <Row type="flex" justify="center" className="mb3" key={localeId}>
           <Col span={6}>
             <Tooltip title={localeTooltipMsg}>
               <a href='#'>{localeId}</a>
             </Tooltip>
           </Col>
           <Col span={6}>
             <Tooltip title={`${srcLang.docCount} Documents`}>
-              <Icon name='document' className='n1' /> <span>
-              {srcLang.docCount} Documents</span>
+              <span className='txt-info'>
+                <Icon name='document' className='n1' />
+                {srcLang.docCount}
```

Comment:
The span used to contain "(docCount) Documents", now it's just docCount. Not sure if that's important. Go ahead and merge if it's not.